backend/core/src/migration/1649260977278-close-overdue-listings.ts

```diff
@@ -0,0 +1,11 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class closeOverdueListings1649260977278 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `UPDATE listings set status = 'closed' where application_due_date < NOW()`
```

Comment:
I think this might publish draft listings into the visible closed section where the due date is in the past, when we want them to stay pending. If this is pulling from the transform removed above, I think we only want to do this where the status is not pending. (Hopefully this state would never happen tho, and the form just handles the status correctly).